src/components/summary/examples/summary-household/index.njk

```diff
@@ -67,3 +67,9 @@
         }
     ]
 }) }}
+
+<div class="u-bt">
```

Comment:
Do we need the two divs? I think what you want can be achieved with adding `u-pt-m` and including the border class on the same div.